Sources/Publish/API/Content.swift

```diff
@@ -17,6 +17,7 @@ public struct Content: Hashable, ContentProtocol {
     public var imagePath: Path?
     public var audio: Audio?
     public var video: Video?
+    public var draft: Bool
```

Comment:
I like this change overall. 👍 I think `isDraft` would make for a more Swifty API.